assets/scss/components/_preview-cart.scss

```diff
@@ -0,0 +1,92 @@
+.c-cart-preview__container {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.c-cart-preview {
+    display: none;
+    position: fixed;
+    right: 0;
+    top: 0;
+    max-width: rem-calc(390px);
+    width: 100%;
+    height: 100vh;
+    background-color: $color-white;
+    z-index: $z-index-very-high;
+    padding: rem-calc(24px);
+    padding-bottom: 0;
+
+    &.loaded {
+        display: block;
+    }
+}
+
+.c-cart-preview__list {
+    overflow-y: auto;
+}
+
+.c-cart-preview__item {
+    flex-direction: column;
+    gap: rem-calc(12px);
+    padding-bottom: rem-calc(24px);
+    margin-bottom: rem-calc(24px);
+    border-bottom: 1px solid $color-secondary-75;
+}
+
+.c-cart-preview__item-information {
+    display: flex;
+    gap: rem-calc(24px);
+
+    .c-cart__item-image-wrapper {
+        max-width: 68px;
+        border-radius: rem-calc(4px);
+        overflow: hidden;
+    }
+}
+
+.c-cart-preview__subtotal-container {
+    display: flex;
+    flex-direction: column;
+    gap: rem-calc(24px);
+    background-color: $color-secondary-50;
+    margin-inline: rem-calc(-24px);
+    padding: rem-calc(24px);
+
+    .c-button {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        gap: rem-calc(8px);
+    }
+}
+
+.c-cart-preview__subtotal {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: rem-calc(24px);
+}
+
+.c-preview-cart__icon {
+    fill: transparent;
+}
+
+.c-cart-preview__subtotal-label,
```

Comment:
Seems like you could have one class `.c-cart-preview__subtotal` (and prevent duplicating the same styles for two different classes) and just add `.u-color-black` / `.u-color-neutrals-300` to adjust color 🤔 

or have something like 

```
.c-cart-preview__subtotal {
   ...

   &--black { ... }
   &--grey { ... } 
}
```